ast/term.go

```diff
@@ -1348,15 +1348,12 @@ func (s *set) String() string {
 	}
 	var b strings.Builder
 	b.WriteRune('{')
```

Comment:
@tsandall sorry f-pushed another one after your review -- this was the change I had not included before 👇